src/ansi-c/library/string.c

```diff
@@ -389,7 +392,7 @@ inline int strcmp(const char *s1, const char *s2)
   }
   while(ch1!=0 && ch2!=0);
   return 0;
-  #endif
+#endif
```

Comment:
Can you please also adjust the matching `#if` to avoid a weirdly looking source?

Reply:
Done.